src/components/InteractionLayer/index.js

```diff
@@ -75,8 +75,9 @@ class InteractionLayer extends React.Component {
     series: [],
     ruler: {
       visible: false,
-      xLabel: () => {},
+      timeLabel: () => {},
       yLabel: () => {},
+      timestamp: null,
```

Comment:
I would imagine at some point we will apply ruler for `ScatterPlot`, so this prop will probably be revisited.